src/library/basesqltablemodel.h

```diff
@@ -127,6 +127,12 @@ class BaseSqlTableModel : public QAbstractTableModel, public TrackModel {
             return order < other.order;
         }
     };
+
+    struct SortColumn {
```

Comment:
Add a 2-arg constructor? That way you can condense 3-lines into 1 in a few places and it won't be possible to construct one that has junk values.